app/assets/stylesheets/common/base/tagging.scss

```diff
@@ -131,6 +131,7 @@
 .discourse-tags {
   display: inline-flex;
   flex-wrap: wrap;
+  position: relative;
```

Comment:
Not against the fix here, but I am guessing this is needed because a child uses `position: absolute`? 

Ah yes, we probably shouldn't use `position: absolute` here: 

https://github.com/discourse/discourse/blob/main/app/assets/stylesheets/common/base/onebox.scss#L783

Reply:
Yes, maybe this: https://github.com/discourse/discourse/pull/19880